common/app/model/dotcomrendering/DotcomRenderingDataModel.scala

```diff
@@ -120,9 +122,11 @@ object DotcomRenderingDataModel {
         "editionLongForm" -> model.editionLongForm,
         "editionId" -> model.editionId,
         "pageId" -> model.pageId,
+        "format" -> model.format,
         "tags" -> model.tags,
         "pillar" -> model.pillar,
         "isImmersive" -> model.isImmersive,
+        "designType" -> model.designType,
```

Comment:
Minor, but should we colocate `format` and `designType` here as they relate?

Reply:
So I moved `designType` further up so that it would be colocated with the other flags that we currently use, but it's likely to be removed in the near future and is technically separate from `Format`. I didn't want them too close in case the `designType` here got mixed up with the `design` part of `Format`. Happy to move it further up if you think that's a sensible idea though